NurVo_Frontend/src/pages/MemberDetails.tsx

```diff
@@ -12,10 +12,13 @@ import MemberDetailCell from '../components/MemberDetailCell';
 import CustomAlert from '../components/Alert';
 
 import img1 from '../assets/images/기본이미지.png';
+import UserContext from '../utilities/UserContext';
+import AsyncStorage from '@react-native-async-storage/async-storage';
 
 const MenberDetails = ({ navigation, route }) => {
   const [userdata, setUserdate] = useState({});
   const [alertOpen, setAlertOpen] = useState('');
+  const { isLogged, setIsLogged } = useContext(UserContext);
 
   useEffect(() => {
     async function getUserData() {
```

Comment:
useEffect(() => {
   async function getUserData() {
```suggestion
      async function getUserData() {
    try {
      const user = await fetchMypage();
      console.log(user);
      setUserdate(user);
    } catch (error) {
      console.error('Error fetching user info:', error);
    }
  }

  useFocusEffect(
    useCallback(() => {
      getUserData();
    }, [])
  );
```
MemberDetails.tsx 의 useEffect 부분을 모두 지우고, 위 코드로 바꿔서 작성해서 수정 사항을 화면에서 바로 확인할 수 있을 거 같아요!